test/test_lang_bind_helper.cpp

```diff
@@ -3135,6 +3135,7 @@ NONCONCURRENT_TEST_IF(LangBindHelper_ImplicitTransactions_InterProcess, !running
     int writepids[write_process_count];
     SHARED_GROUP_TEST_PATH(path);
 
+    clear_mappings_before_test_forks();
```

Comment:
I was thinking we needed to worry about the other forks, but, since the read and write operations will happen in other processes, they shouldn't affect the reclaimer in the current (main) process.